test_schemas/en/test_checkbox.json

```diff
@@ -32,6 +32,67 @@
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "null-label-checkbox-answer",
+                                        "mandatory": false,
+                                        "label": null,
+                                        "options": [
+                                            {
+                                                "label": "None",
+                                                "value": "None",
+                                                "q_code": "0"
+                                            },
+                                            {
+                                                "label": "Cheese",
+                                                "value": "Cheese",
+                                                "q_code": "1"
+                                            },
+                                            {
+                                                "label": "Ham",
+                                                "value": "Ham",
+                                                "q_code": "2"
+                                            },
+                                            {
+                                                "label": "Pineapple",
+                                                "value": "Pineapple",
+                                                "q_code": "3"
+                                            },
+                                            {
+                                                "label": "Tuna",
+                                                "value": "Tuna",
+                                                "q_code": "4"
+                                            },
+                                            {
+                                                "label": "Pepperoni",
+                                                "value": "Pepperoni",
+                                                "q_code": "5"
+                                            },
+                                            {
+                                                "label": "Other",
+                                                "q_code": "6",
+                                                "description": "Choose any other topping",
+                                                "value": "Other",
+                                                "detail_answer": {
+                                                    "mandatory": true,
+                                                    "id": "null-label-other-answer-mandatory",
+                                                    "label": "Please specify other",
+                                                    "type": "TextField"
+                                                }
+                                            }
```

Comment:
Test schemas should be as small as possible to focus on what is being tested:
- There is no need for this many options when testing for no label on the answer, 2 or 3 is enough
- `q_code` is optional, so should be removed (I can see this is the same for existing questions in this test, but they can be removed too).